packages/editor/src/browser/editor-keybinding.ts

```diff
@@ -41,6 +41,10 @@ export class EditorKeybindingContribution implements KeybindingContribution {
                 command: EditorCommands.TOGGLE_WORD_WRAP.id,
                 keybinding: 'alt+z'
             },
+            {
+                command: EditorCommands.TOGGLE_STICKY_SCROLL.id,
+                keybinding: ''
+            },
```

Comment:
The following can be removed, I don't believe the sticky scroll command has a keybinding.

Reply:
Well noted, will do. Many thanks!